app/Jobs/ImportRockTheVoteReport.php

```diff
@@ -72,4 +72,17 @@ public function handle()
         $this->report->dispatched_at = $now;
         $this->report->save();
     }
+
+    /**
+     * Returns the parameters passed to this job.
+     *
+     * @return array
+     */
+    public function getParameters()
+    {
+        return [
+            'report' => $this->report,
+            'user' => $this->user,
```

Comment:
Should we return `user_id` here rather than the full User model?

Reply:
I went with a full User model because that's what we're expecting when we [dispatch a `ImportFileRecords` job](https://github.com/DoSomething/chompy/blob/674ad32ebffcc0bec959f1d63e4fb69fa3bb9579/app/Jobs/ImportFileRecords.php#L51) if the report status is complete.